wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,75 @@
+
+ 1. Go to your favourite searcher and search: “{your brand device name} drivers”.
+ 2. Next, put the ID of your device and follow the instructions on the page.
+ 3. When you selected the product you have two options:
+	- Download the driver downloader (recomended)
+	- And download the driver on especific and run executable
+
+## Method 3: Download via brand product page.
+Now if you want to download for your GPU or your brand device hasn't the drivers (in this case do it if it is especially important, otherwise apply the saying: “if it works, don't touch it, or you may make it worse”).
+Now follow the same instructions but the brand hardware (like intel CPU and GPU).
+
+ 1. Go to your favourite searcher again and search: “{your brand name} driver download”.
+ 2. Search your hardware piece you want to upgrade.
+ 3. When you selected the product you have two options:
+	- Download the driver downloader (recomended)
+	- And download the driver on especific and run executable
+- (Follow method 3 for more information: [https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website))
+
+## Oh, no! I can't access to my pc!
+
+1. Enter Safe Mode:
+
+Safe mode is a diagnostic boot way that loads Windows with only the essential drivers. This allows you to access Windows even if the problematic driver is causing issues during normal boot.
+
+ - Windows 10/11: Repeatedly press and hold the power button while booting to trigger a forced restart. After 2–3 restarts, you'll see the “Advanced options” screen. Navigate to “Troubleshoot” > “Advanced options” > “Startup Settings” > “Restart now”. Upon restart, choose “Enable Safe Mode with Networking” (if you need internet) or simply “Safe Mode”.
+
+ - Earlier Windows versions: Restart your computer and repeatedly press F8 before the Windows logo appears. This should open the Advanced Boot Options menu, where you can choose “Safe Mode”.
+
+2. Uninstall the Driver in Safe Mode:
+
+Once in safe mode, you can uninstall the problematic driver:
+
+ 1. Open the Device Manager (search for it in the Start Menu).
+ 2. Expand the category for the device with the recently installed driver.
+ 3. Right-click on the driver and select “Uninstall device”.
+ 4. Check the box “Delete the driver software for this device” (if available) and click “Uninstall”.
+
+3. System Restore (if applicable):
+
+If you recently created a system restore point before installing the driver, you can restore Windows to that point:
+
+ 1. In Safe Mode, open the Start Menu and search for “System Restore”.
+ 2. Click “Create a restore point” and then “Next”.
+ 3. Choose the desired restore point (ideally the one before installing the driver) and click “Next” followed by “Finish”.
+
+4. Startup Repair (if Windows won't boot at all):
+
+If even Safe Mode isn't accessible, you can try Startup Repair, a built-in tool on Windows that tries to fix boot issues:
```

Comment:
If Safe Mode isn't accessible, you can try Startup Repair, a built-in tool **in** Windows that will attempt to fix boot issues: